package.json

```diff
@@ -58,7 +58,9 @@
     "pre-commit": "^1.2.2",
     "prettier": "^1.15.1",
     "typescript": "^3.0.3",
-    "typescript-eslint-parser": "^18.0.0"
+    "typescript-eslint-parser": "^18.0.0",
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
why put on the top package, alongside babel-loader? They are not used on more than one package, are they?

Reply:
they are

Reply:
when using in more than one package, just add to the root.

Reply:
monorepo "bons costumes"

Reply:
interesting... i already thought the same. @sibelius what do you think? Personally i think having one source of truth and making it easy to update dependencies is a :heavy_plus_sign:

Reply:
there are more duplicated deps.. and babel package should be a `babel-preset`.. so u can use in ur babel config.

Check: https://github.com/FotonTech/gigatron